### PR TITLE
[IMP] Wizard template

### PR DESCRIPTION
**Pull request by adrienpeiffer**
- [IMP] wizard : use new target in wizard action
- [ADD] wizard : add generation of multi action for wizard template

### CODE REVIEW

**Comment by adrienpeiffer** [comment]
@sbidoul je ne comprends pas vraiment pourquoi tox est rouge ...

**Comment by adrienpeiffer** [comment]
Rebased

File: bobtemplates/odoo/wizard/wizards/+wizard.name_underscored+.xml.bob [hunk 1]
```diff
@@ -43,6 +43,16 @@
...
+        <field name="target">new</field>
+    </record>
+{{% endif %}}
+
+{{% if wizard.action_multi %}}
+    <record model="ir.values" id={{{ wizard.name_underscored }}}_act_multi">
+        <field name="name">{{{ wizard.name_camelwords }}}</field><!-- TODO -->
+        <field name="key2">client_action_multi</field>
+        <field name="value" eval="'ir.actions.act_window,' +str(ref('{{{ wizard.name_underscored }}}_act_window'))" />
+        <field name="model"></field><!-- TODO -->
```
Comment (sbidoul) [inline bobtemplates/odoo/wizard/wizards/+wizard.name_underscored+.xml.bob]: indicate src.model or something so it's easier to understand what to put here

Comment (adrienpeiffer) [inline bobtemplates/odoo/wizard/wizards/+wizard.name_underscored+.xml.bob]: Good idea

Comment (adrienpeiffer) [inline bobtemplates/odoo/wizard/wizards/+wizard.name_underscored+.xml.bob]: Done !

File: bobtemplates/odoo/wizard/.mrbob.ini [hunk 2]
```diff
@@ -14,6 +14,10 @@ wizard.view_action.question = Wizard action
 wizard.view_action.default = y
 wizard.view_action.post_ask_question = mrbob.hooks:to_boolean
 
+wizard.action_multi.question = Multi action
```
Comment (sbidoul) [inline bobtemplates/odoo/wizard/.mrbob.ini]: Wizard action in the Action/More menu

**Comment by sbidoul** [comment]
I think tests are red because you must add the answer to the new question in https://github.com/acsone/bobtemplates.odoo/blob/master/tests/test_odoo_wizard_answers.ini

**Comment by adrienpeiffer** [comment]
Strange, I was sure I had done.
Thanks

**Comment by adrienpeiffer** [comment]
CI is happy :smile:

**Comment by sbidoul** [comment]
Cool, thanks!

**Pull request merged**